lib/src/api/player/player_config.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:bitmovin_player/bitmovin_player.dart';
+import 'package:bitmovin_player/src/api/casting/remote_control_config.dart';
```

Comment:
We can remove this import as it is included in 
```
import 'package:bitmovin_player/bitmovin_player.dart';
```

Reply:
```suggestion
```